keras/layers/multi_head_attention.py

```diff
@@ -195,7 +195,7 @@ class MultiHeadAttention(Layer):
       indicates no attention. Broadcasting can happen for the missing batch
       dimensions and the head dimension.
     return_attention_scores: A boolean to indicate whether the output should
-      be attention output if True, or (attention_output, attention_scores) if
+      be (attention_output, attention_scores) if True, or attention_output if
```

Comment:
Use backticks around code keywords, e.g. `(attention_output, attention_scores)` and `attention_output`

Reply:
Missed that, it should be fine now. Thanks!